pkg/assetfetcher/http_test.go

```diff
@@ -629,8 +628,56 @@ func TestMakeRequestSuccess(t *testing.T) {
 
 	wg.Add(1)
 	nexposeAssetFetcher.makeRequest(context.Background(), &wg, "siteID", 100, assetChan, errChan)
+	wg.Wait()
+	assert.NotNil(t, <-assetChan)
```

Comment:
suggestion: In the event that the method exits but with an error instead of an asset then you'll hit another timeout condition here since reading from the empty channel will block forever. If you don't particularly care what the contents are and only that there was an element written to one channel or the other then you might consider using `assert.Len(t, 1, assetChan)` since the `len()` of a channel is the number of elements waiting to be read. 

ex: https://play.golang.org/p/9ZLu69mkHdb

In these cases, I also recommend asserting both conditions: `Len(t, 1, assetChan)` and `Len(t, 0, errChan)` to cover more edge cases.

Reply:
done! seems like assert has a lot of different functions to assert on